docs/ref/middleware.txt

```diff
@@ -155,6 +155,9 @@ It will NOT compress content if any of the following are true:
 * The request (the browser) hasn't sent an ``Accept-Encoding`` header
   containing ``gzip``.
 
+If the response has an ``ETag`` header, the header will be changed to a weak
+``ETag`` to comply with :rfc:`7232`.
```

Comment:
Good idea, I think https://tools.ietf.org/html/rfc7232#section-2.1 covers it. (Could be used in the comment as well.)